sponsor/views.py

```diff
@@ -1,12 +1,12 @@
-from django.views.generic.base import TemplateView
-
-from core.utils import next_deadline
-
-
-class SponsorRequestView(TemplateView):
-    template_name = "sponsor/sponsor-request.html"
-
-    def get_context_data(self, **kwargs):
-        context = super(SponsorRequestView, self).get_context_data(**kwargs)
-        context["deadline"] = next_deadline()
-        return context
+# from django.views.generic.base import TemplateView
+#
+# from core.utils import next_deadline
+#
+#
+# class SponsorRequestView(TemplateView):
+#     template_name = "sponsor/sponsor-request.html"
+#
+#     def get_context_data(self, **kwargs):
+#         context = super(SponsorRequestView, self).get_context_data(**kwargs)
+#         context["deadline"] = next_deadline()
+#         return context
```

Comment:
Yeah as above, it's not loaded by the site, but does get loaded by coverage which says it's not tested, but it can't be tested because it's not loaded.